src/nvim/mark.c

```diff
@@ -922,17 +922,33 @@ void ex_delmarks(exarg_T *eap)
   int lower;
   int digit;
   int n;
+  const Timestamp timestamp = os_time();
```

Comment:
I think `os_time()` should be called in the two branches where it is used separately. It doesn't need to be called when there is an error.